### PR TITLE
Don't generated type definitions for aliased types

### PR DESCRIPTION
**Pull request by jtaylorX2**

### CODE REVIEW

File: src/Ccap/Codegen/Database.purs [hunk 1]
```diff
@@ -52,7 +60,14 @@ domainModule pool scalaPkg pursPkg =
         results <- query conn (Query sql) Row0
         let
           types = Array.sortWith Cst.typeDeclName $ domainTypeDecl <<< rowToDomain <$> results
-        nelTypes <- except ((note (ConversionError "Expected at least one type")) (NonEmptyArray.fromArray types))
+
+          typesWithoutAlias =
+            Array.filter
+              ( \(Cst.TypeDecl typeDecl) ->
+                  isNothing (Array.find (\aliasedType -> typeDecl.name == aliasedType.name) aliasedTypes)
```
Comment (pbrant) [inline src/Ccap/Codegen/Database.purs]: `not any`

File: src/Ccap/Codegen/Database.purs [hunk 2]
```diff
@@ -179,4 +194,4 @@ dbType dataType = case dataType of
   "uuid" -> Cst.Ref emptyPos { mod: Just (Cst.ModuleRef "UUIDSupport"), typ: "UUID", params: [] }
   "interval" -> Cst.Ref emptyPos { mod: Just (Cst.ModuleRef "DateTimeSupport"), typ: "Duration", params: [] }
   "occid" -> Cst.Ref emptyPos { mod: Just (Cst.ModuleRef "OccSupport"), typ: "OccId", params: [] }
-  _ -> Cst.Primitive Cst.PString -- XXX
+  _ -> Cst.Primitive Cst.PString -- XXX
```
Comment (pbrant) [inline src/Ccap/Codegen/Database.purs]: Could you revert this change?

**Review by pbrant — CHANGES_REQUESTED** [review]

File: src/Ccap/Codegen/Database.purs (hunk 2 above)
Comment (jtaylorX2) [reply src/Ccap/Codegen/Database.purs]: Sure thing!
